x-pack/plugins/apm/server/plugin.ts

```diff
@@ -161,6 +145,28 @@ export class APMPlugin implements Plugin<APMPluginSetup> {
         config: await mergedConfig$.pipe(take(1)).toPromise(),
       });
 
+    const apmRuleRegistry = plugins.observability.registry.create({
+      namespace: 'apm',
+      fieldMap: {
```

Comment:
Does `service.name` go here?

Reply:
oic it's in the observability registry. The inheritance is nice but if you're just looking at the code you would have to walk the tree to grok the fieldmap, at least in your head. Hopefully good types will mitigate that.

Reply:
The types are magic 👽